website/source/docs/platform/k8s/helm/configuration.html.md

```diff
@@ -85,7 +127,7 @@ and consider if they're appropriate for your deployment.
 
     * `authDelegator` - Values that configure the Cluster Role Binding attached to the Vault service account.
 
-        - `enabled` (`boolean: false`) - When set to `true`, a Cluster Role Binding will be bound to the Vault service account.  This Cluster Role Binding has the necessary privileges for Vault to use the [Kubernetes Auth Method](/docs/auth/kubernetes.html).
+        - `enabled` (`boolean: true`) - When set to `true`, a Cluster Role Binding will be bound to the Vault service account.  This Cluster Role Binding has the necessary privileges for Vault to use the [Kubernetes Auth Method](/docs/auth/kubernetes.html).
```

Comment:
Same question about what the default is here.

Reply:
Yes, this is the default.